### PR TITLE
Manage Customer Feature Added

### PR DESCRIPTION
**Pull request by antochris00**
Manage Customer feature Added in Admin UI and made changes in Client UI

### CODE REVIEW

File: admin-ui/src/app/views/employee/customer/customer.component.scss [hunk 1]
```diff
@@ -0,0 +1,23 @@
+// ::ng-deep .mat-slide-toggle-bar {
```
Comment (muhilkennedy) [inline admin-ui/src/app/views/employee/customer/customer.component.scss:1]: Remove commented lines unless necessary

File: backendParent/api/src/main/java/com/backend/api/admin/controller/CustomerAdminController.java [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+import com.backend.api.messages.GenericResponse;
+import com.backend.api.messages.Response;
+import com.backend.core.entity.EmployeeInfo;
+import com.backend.core.service.BaseService;
+import com.backend.core.util.Constants;
+import com.backend.persistence.entity.CustomerAddress;
+import com.backend.persistence.entity.CustomerInfo;
+import com.backend.persistence.service.CustomerInfoService;
+import com.backend.persistence.service.EmployeeService;
+
```
Comment (muhilkennedy) [inline backendParent/api/src/main/java/com/backend/api/admin/controller/CustomerAdminController.java:34]: Use @author annotation

File: backendParent/api/src/main/java/com/backend/api/admin/controller/CustomerAdminController.java [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+/*By Chris
+ * 
+ */
+
+@RestController
+
+//when 'manage customer' is clicked from admin UI it goes through the Realm,token and admin filters for original admin validation and only after then the corresponding request mapping is called
```
Comment (muhilkennedy) [inline backendParent/api/src/main/java/com/backend/api/admin/controller/CustomerAdminController.java:41]: Remove this comment. This should be specific to this class only!

File: backendParent/persistence/src/main/java/com/backend/persistence/repository/CustomerInfoRepository.java [hunk 4]
```diff
@@ -31,4 +33,19 @@
 	
 	@Query(findCustomerByEmailOrMobileQuery)
 	CustomerInfo findCustomerByEmailOrMobile(@Param("email") String emailOrMobile, @Param("tenant") Tenant realm);
+	
+	/* Author - Chris */
```
Comment (muhilkennedy) [inline backendParent/persistence/src/main/java/com/backend/persistence/repository/CustomerInfoRepository.java:37]: Remove this comment

File: backendParent/persistence/src/main/java/com/backend/persistence/serviceImpl/CustomerInfoServiceImpl.java [hunk 5]
```diff
@@ -169,4 +169,27 @@ public void updateCustomerMobile(String mobile) throws Exception {
 		}
 	}
```
Comment (muhilkennedy) [inline backendParent/persistence/src/main/java/com/backend/persistence/serviceImpl/CustomerInfoServiceImpl.java:171]: Remove comment

**Review by muhilkennedy — CHANGES_REQUESTED** [review]
Pls address the comments

File: admin-ui/src/app/views/employee/customer/customer.component.scss (hunk 1 above)
Comment (antochris00) [reply admin-ui/src/app/views/employee/customer/customer.component.scss:1]: Left the CSS commented to change the size of mat-slide-toggle-bar in future

**Review by muhilkennedy — APPROVED** [review]

**Pull request merged**